demo/universal/native/lib/ServerFunctions.re

```diff
@@ -309,8 +347,74 @@ module Samples = {
       )
     };
 
+  // Lets say this is the server action declared by the end-user
+  /**
+    [@react.server.action]
+    let simpleResponse = () => {
+      Lwt.return(
+        (`String("Hello from server with simple response action")),
+      );
+    };
+  */
+  // It's going to be on top to this that we are going to generate the codes bellow
+  let errorId = "id/samples/error";
+
+  [@platform native]
+  let error = () => {
+    let stack = createStackError();
+
+    failwith(Printf.sprintf("Error from server"));
+  };
+
+  // This is the router  handler that will handle parsing the args and calling the handler the user declared
+  // This code will be generated by the ppx automatically
+  [@platform native]
+  let errorRouteHandler = _args =>
+    try(error() |> Lwt.map(response => response)) {
+    | e =>
+      let msg = Printexc.to_string(e);
+      Lwt.return(
+        React.Error({
```

Comment:
React.Error is the client_value for error, like we have for React.Json: https://github.com/ml-in-barcelona/server-reason-react/pull/247/files#diff-20b5f5f821cea6aaaa4897da4fdbf5bc8d0c624ad6e2c609294a188f2fb9202dR318-R320